setup.py

```diff
@@ -859,7 +859,11 @@ def debug_build():
         license="HPND",
         author="Alex Clark (PIL Fork Author)",
         author_email="aclark@python-pillow.org",
-        url="http://python-pillow.org",
+        url="https://python-pillow.org/",
+        project_urls={
+            "Documentation": "https://pillow.readthedocs.io/en/stable/",
+            "Source Code": "https://github.com/python-pillow/Pillow",
```

Comment:
Let's use the [more common](https://github.com/hugovk/pypi-tools/issues/11) option:
```suggestion
            "Source": "https://github.com/python-pillow/Pillow",
```